server/@types/models/Referral.ts

```diff
@@ -6,6 +6,7 @@ type Referral = {
   oasysConfirmed: boolean
   offeringId: CourseOffering['id']
   prisonNumber: Person['prisonNumber']
+  reason: string
```

Comment:
Technically the Nunjucks doesn't need to typecheck, but I wonder if this should go before it's used in the template (currently the first commit)